### PR TITLE
Test ride request 158655293

### PR DESCRIPTION
**Pull request by Ogollah**
#### What does this Pr do?
creates ride offer requests test cases
#### Description of Task to be completed?
Expects the test cases for the following end points to fail
`GET/<user_id>/ride/<ride_id>/requests `
 `GET/<user_id>/ride/requests/<request_id>`
 `POST/<user_id>/ride/requests/<request_id>/respond` 
#### How should this be manually tested?
`usingterminal`
 [clone this repo](https://github.com/Ogollah/Ride-my-way-api.git)
`cd Ride-my-way-api`
`set and activate your virtual environment`
`git checkout test-ride-request`
`Run nosetests`
### What are the relevant pivotal tracker stories?
[#158655293](https://www.pivotaltracker.com/story/show/158655293)

### CODE REVIEW

File: tests/test_ride.py [hunk 1]
```diff
@@ -192,15 +190,13 @@ def test_already_created_ride_offer(self):
...
+        self.client().post('/api/v1/auth/signup', data=self.user_data_2)
         login_response = self.client().post('/api/v1/auth/login', data=self.user_data_2)
-        self.assertEqual(login_response.status_code, 200)
         #Define header dictionary
         access_token = json.loads(login_response.data.decode())['access_token']
         #create a new ride offer
-        response = self.client().post('/api/v1/ride/create',
-                                      headers=dict(Authorization='Bearer ' + access_token), data=self.ride_data_1)
+        self.client().post('/api/v1/ride/create',
+                           headers=dict(Authorization='Bearer ' + access_token), data=self.ride_data_1)
```
Comment (meshack-mbuvi) [inline tests/test_ride.py:199]: You can set the authorization header on the setup method instead of having to set it here.

Comment (Ogollah) [reply tests/test_ride.py:199]: Thanks let me try that

**Comment by xcixor** [comment]
Your code @Ogollah  could be more readable, check on line length

**Comment by Ogollah** [comment]
@xcixor Noted

**Pull request merged**